### PR TITLE
Added ask only once

### PR DESCRIPTION
**Pull request by Qubasa**

### CODE REVIEW

File: find-libs [hunk 1]
```diff
@@ -1,12 +1,36 @@
 #!/bin/sh
-LIBS_NOT_FOUND="$(ldd $1 | grep 'not found' | sed 's/ .*//;s/\t//' | sort | uniq)"
+LIBS_NOT_FOUND="$(ldd "$1" | grep 'not found' | sed 's/ .*//;s/\t//' | sort | uniq)"
+SELECTED_LIBS=""
 
+# Iterate over every dependencie of binary
 for x in $LIBS_NOT_FOUND; do
-  candidates=$(nix-locate -w --top-level $x)
+    candidates="$(nix-locate -w --top-level "$x")"
```
Comment (Lassulus) [inline find-libs]: quotes around $(...) are unnecessary on assignment

File: find-libs [hunk 2]
```diff
@@ -1,12 +1,36 @@
 #!/bin/sh
-LIBS_NOT_FOUND="$(ldd $1 | grep 'not found' | sed 's/ .*//;s/\t//' | sort | uniq)"
+LIBS_NOT_FOUND="$(ldd "$1" | grep 'not found' | sed 's/ .*//;s/\t//' | sort | uniq)"
```
Comment (Lassulus) [inline find-libs]: quotes around $(...) are unnecessary on assignments

File: find-libs (hunk 1 above)
Comment (Lassulus) [inline find-libs]: please use 2 spaces indent (style conformity)

File: nix-autobahn [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
-set -x
-export SCRIPT=$(realpath "$1")
-SCRIPTDIR="$(dirname $0)"
-LIBS=$($SCRIPTDIR/find-libs $SCRIPT)
+# set -x
+export SCRIPT
+SCRIPT=$(realpath "$1")
 
-$SCRIPTDIR/fhs-shell $LIBS
+SCRIPTDIR="$(dirname "$0")"
```
Comment (Lassulus) [inline nix-autobahn]: no quotes around $(..) on assignments

File: nix-autobahn [hunk 4]
```diff
@@ -1,7 +1,9 @@
 #!/bin/sh
-set -x
-export SCRIPT=$(realpath "$1")
-SCRIPTDIR="$(dirname $0)"
-LIBS=$($SCRIPTDIR/find-libs $SCRIPT)
+# set -x
```
Comment (Lassulus) [inline nix-autobahn]: please remove dead code

**Review by Lassulus — CHANGES_REQUESTED** [review]

**Review by Qubasa — COMMENTED** [review]
Fixed

**Pull request merged**